packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -104,6 +137,72 @@ const modifiers = [
   },
 ];
 
+const ListButton = ({ icon, format, label }) => {
+  const editor = useSlate();
+
+  const isListActive = () => {
+    const { selection } = editor;
+
+    if (!selection) return false;
+
+    const [match] = Array.from(
+      Editor.nodes(editor, {
+        at: Editor.unhangRange(editor, selection),
+        match: (node) =>
+          !Editor.isEditor(node) &&
+          Element.isElement(node) &&
+          node.type === 'list' &&
+          node.format === format,
+      })
+    );
+
+    return !!match;
```

Comment:
Another code style comment, I just want to have the discussion you don't have to change if your prefer this:

I think about this one every time I see it. Why use a double negation when `Boolean(value)` is available? The way I see it the double negation looks like code while the Boolean looks like english. I like code but I still prefer english. Is there a difference between the two I'm missing 🤷‍♂️